docs/DeveloperGuide.md

```diff
@@ -834,3 +834,6 @@ Common prerequisite: List all elderly using the `list` command.
    1. Delete the `data/contactmate.json` file.
    2. Launch the app.<br>
        Expected: The app should show a list of sample persons.
+
+3. **Mark elderly as called in the history view:** Currently, users can view the call history and profile view of the elderly in the history view together. This allows users to view the NRIC of the elderly, which may give the wrong impression that they can mark the elderly using NRIC as called in the history view (e.g. mark SXXXXXXXH). However, this is unintended as the history view is meant for viewing purposes only. We plan to provide the ability to mark the elderly whose details is specified in the history view as called.
+4. **Maintain the current filtered person list and index position after editing or adding an elderly contact:** If a user used the `find` command to filter the person list before editing or adding an elderly contact, the person list will reset to show all elderly. We plan to maintain the current filtered person list and index position after editing. Also, any added elderly will be appended to the back of the filtered list to maintain index position.
```

Comment:
Similar to my comment above, can frame it as  -> Maintain the current filtered person list and index position after modification of the elderly list

then it encapsulates mark also

another benefit of doing this instead of what you did is that I THINK it makes it clear that its only one enhancement, whereas you saying 

> after editing or adding an elderly contact:

might be considered 2 if they really nitpicky

Reply:
also what you mean by maintaining index position

Reply:
again good idea on the modification.

maintaining index position like if the current list is 
John doe
John lim
John ng

and I edit John Lim to be John X, then I want the position of previously John Lim to stay at the same index.

Any suggestions for a better way to phrase this?

Reply:
Probably the order of the list?

But, in the spirit of trying to make it sound like only ONE enhancement, can combine order and filter to one word, maybe like `maintain the state of the list after modification of the elderly list`

If so, legit need elaborate a bit, but rmb make it sound like one enhancement only